ingestion/examples/workflows/athena.yaml

```diff
@@ -4,6 +4,7 @@ source:
   serviceConnection:
     config:
       type: Athena
+      supportsQueryComment: false
```

Comment:
I think we should only add this property to the connections which support query comments and then check in the python code like this:
```python
 if hasattr(self.config.source.serviceConnection.__root__.config, "supportsQueryComment")
```
at least is how we are checking other `support` properties for example with the [supportDatabase](https://github.com/open-metadata/OpenMetadata/blob/7863f040d1486d7920d0e4293631d973d66fd7b2/ingestion/src/metadata/orm_profiler/api/workflow.py#L357) one.

Reply:
Okay, sure@nahuelverdugo.